crypto/dh/dh_kdf.c

```diff
@@ -185,3 +190,4 @@ int DH_KDF_X9_42(unsigned char *out, size_t outlen,
     EVP_MD_CTX_cleanup(&mctx);
     return rv;
 }
+#endif
```

Comment:
Ok, so I applied this patch:

``` diff
diff --git a/crypto/ebcdic.c b/crypto/ebcdic.c
index fd6df92b46..2489e4df51 100644
--- a/crypto/ebcdic.c
+++ b/crypto/ebcdic.c
@@ -3,9 +3,11 @@
 #ifndef CHARSET_EBCDIC
 
 # include <openssl/e_os2.h>
+# if 0
 # if defined(PEDANTIC) || defined(__DECC) || defined(OPENSSL_SYS_MACOSX) || defined(__clang__)
 static void *dummy = &dummy;
 # endif
+# endif
 
 #else                           /* CHARSET_EBCDIC */
 
```

And then configured and built:

```
: ; make clean; CC=clang ./config --strict-warnings
: ; make
```

.... and eventually, it ended with this:

```
clang -I. -I.. -I../include  -DOPENSSL_THREADS -D_REENTRANT -DDSO_DLFCN -DHAVE_DLFCN_H -m64 -DL_ENDIAN -O3 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wno-language-extension-token  -Wno-extended-offsetof -Qunused-arguments -DOPENSSL_IA32_SSE2 -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_BN_ASM_GF2m -DRC4_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DMD5_ASM -DAES_ASM -DVPAES_ASM -DBSAES_ASM -DWHIRLPOOL_ASM -DGHASH_ASM -DECP_NISTZ256_ASM -pedantic -DPEDANTIC -Wno-long-long -Wsign-compare -Wmissing-prototypes -Wshadow -Wformat -Wundef -Werror -DCRYPTO_MDEBUG_ALL -DCRYPTO_MDEBUG_ABORT -DREF_CHECK -DOPENSSL_NO_DEPRECATED   -c -o ebcdic.o ebcdic.c
ebcdic.c:286:7: error: ISO C requires a translation unit to contain at least one
      declaration [-Werror,-Wempty-translation-unit]
#endif
      ^
1 error generated.
```